wiki/Legal/SCTL/en.md

```diff
@@ -0,0 +1,37 @@
+---
+legal: true
+---
+
+# The following information is disclosed based on Article 11 of the "Act on Specified Commercial Transactions"
+
+## Name of the seller:
+
+ppy Pty Ltd ACN 163 593 413 a.t.f. Dean Herbert Family Trust
+
+## Location of the seller:
+
+41 Gregory Street, Wembley, WA, 6014, Australia
+
+## Manager of the seller:
+
+Director: Dean Herbert
+
+## Sales price:
+
+Please check the purchase page on the site.
+
+## Payment methods:
+
+You can use any of the following payment methods:
+
+Credit card, cash payment, Paypal, Docomo, NetCash, BitCash
+
+## Cancellation:
+
+We accept cancellation and refund requests per customer's discretion. Please contact accounts@ppy.sh.
+
+## Contact:
+
+https://osu.ppy.sh/wiki/ja/Main_page
```

Comment:
why is the wiki main page a "contact" link?

Reply:
I'm mostly matching other existing example pages, which link to a game help page.

Reply:
I've added a prefix to give context.